templates/controllers/grid/pubIds/pubIdExportSubmissionsGridFilter.tpl

```diff
@@ -0,0 +1,32 @@
+{**
+ * templates/controllers/grid/pubIds/pubIdExportSubmissionsGridFilter.tpl
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2000-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * Filter template for submissions lists.
+ *}
+{assign var="formId" value="submissionsListFilter-"|concat:$filterData.gridId}
```

Comment:
Hmm, I wonder if there's not a cleaner way of exposing the right ID to the templates.